tests/Unit/PrestaShopBundle/Form/Validator/TinyMceMaxLengthValidatorTest.php

```diff
@@ -0,0 +1,178 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Unit\PrestaShopBundle\Form\Validator;
+
+use Exception;
+use InvalidArgumentException;
+use PrestaShop\PrestaShop\Adapter\Validate;
+use PrestaShopBundle\Form\Validator\Constraints\TinyMceMaxLength;
+use PrestaShopBundle\Form\Validator\Constraints\TinyMceMaxLengthValidator;
+use stdClass;
+use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Validator\Exception\MissingOptionsException;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+
+class TinyMceMaxLengthValidatorTest extends ConstraintValidatorTestCase
+{
+    protected function createValidator(): TinyMceMaxLengthValidator
+    {
+        $this->translator = $this->createMock(TranslatorInterface::class);
+
+        return new TinyMceMaxLengthValidator(new Validate(), $this->translator);
+    }
+
+    /**
+     * @param mixed $max
+     * @param string|null $message
+     *
+     * @return TinyMceMaxLength
+     */
+    private function getConstraintInstance($max, ?string $message = null): TinyMceMaxLength
+    {
+        return new TinyMceMaxLength([
+            'max' => $max,
+            'message' => $message,
+        ]);
+    }
+
+    /**
+     * @param Exception $expectedException
+     * @param mixed $tinyMceMaxLength
+     *
+     * @dataProvider exceptionsInvalidTinyMceMaxLengthProvider
+     */
+    public function testExceptionsInvalidTinyMceMaxLength(Exception $expectedException, $tinyMceMaxLength): void
+    {
+        $expectedExceptionClassName = \get_class($expectedException);
```

Comment:
```suggestion
        $expectedExceptionClassName = get_class($expectedException);
```

Reply:
compile time inlining 
See https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3048 by @nicolas-grekas

Reply:
In this case, move to use `use function get_class`.

Reply:
This isn't something we implement yet :thinking:

Reply:
what should I do ?